callmetadata.schema.json

```diff
@@ -7,27 +7,46 @@
     "definitions": {
         "participant": {
             "type": "object",
-            "description": "Specific details about the farEndUsers (the other end of the call)",
+            "description": "Specific details about a participant of the call",
+            "additionalProperties": true,
             "properties": {
-                "firmName": {
+                "name": {
                     "type": "string",
-                    "example": "XYZ Firm",
-                    "description": "Name of the farEnd firm on the call. If an internal call, the firmName can match the nearEnd firm"
+                    "example": "Steve Jones",
+                    "description": "Name of the participant on the call"
                 },
-                "groupId": {
```

Comment:
Removed groupId information from the standard. System internal ids are not metadata information and we need to focus on metadata information and allow vendors to extend if they want to store vendor specific information.